src/Console/Command/CleanMySQLDatabaseCommand.php

```diff
@@ -0,0 +1,178 @@
+
+use JiraRestApi\Configuration\ArrayConfiguration;
+use JiraRestApi\Issue\IssueService;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Finder\SplFileInfo;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * Class CleanMySQLDatabaseCommand
+ *
+ * @package duckpony\Console\Command
+ * @author  Michel Petiton <michel.petiton@check24.de>
+ */
+class CleanMySQLDatabaseCommand extends AbstractCommand
+{
+    private const DATABASE_BLACKLIST = [
+        'information_schema',
+        'perfomance_schema',
+        'mysql',
+    ];
+
+    /**
+     * Configures the console command
+     *
+     * @return void
+     */
+    protected function configure()
+    {
+        $this->addOption('status', 's', InputOption::VALUE_REQUIRED, 'Status');
+        $this->addOption('pattern', 'p', InputOption::VALUE_REQUIRED, 'Branch pattern');
+        $this->addOption('invert', 'i', InputOption::VALUE_NONE, 'Invert status');
+        $this->addOption('config', 'c', InputOption::VALUE_REQUIRED, 'Config', $this->getRootPath() . '/config/config.yml');
+        $this->addArgument('branchname-filter', InputArgument::IS_ARRAY | InputArgument::OPTIONAL , 'Filter branchname');
+
+        $this->setName('db:clean')
+             ->setDescription('Scans Database and cleans orphaned')
+             ->setHelp(
+                 <<<EOT
+Scans MySQL Databases and removes
+them under certain conditions
+EOT
+             );
+    }
+
+    /**
+     * Executes the console command
+     *
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return int|null|void
+     * @throws \JiraRestApi\JiraException
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $statuses = $this->fetchStatuses($input->getOption('status'));
+        $invert = $input->getOption('invert');
+        $config = Yaml::parse(file_get_contents($input->getOption('config')));
+        $pattern = $input->getOption('pattern');
+        $branchNameFilter = $input->getArgument('branchname-filter');
+
+        if (!isset(
+            $config['MySQL']['username'],
+            $config['MySQL']['password'],
+            $config['MySQL']['hostname']
+        )) {
+            throw new \RuntimeException('MySQL config is incorrect! Username password and hostname required!');
+        }
+
+        $dbUser = $config['MySQL']['username'];
+        $dbPassword = $config['MySQL']['password'];
+        $dbHost = $config['MySQL']['hostname'];
+
+        $issueService = new IssueService(new ArrayConfiguration([
+            'jiraHost' => $config['CleanBranch']['hostname'],
```

Comment:
Config 'CleanBranch' should be more revealing as it is used in both, CleanBranch and CleanMySQL Commands.